agent/agent.go

```diff
@@ -304,19 +304,25 @@ func (a *Agent) endTaskResponse(tc *taskContext, status string) *apimodels.TaskE
 }
 
 func (a *Agent) runPostTaskCommands(ctx context.Context, tc *taskContext) {
+	var err error
+	start := time.Now()
 	if tc.taskConfig != nil && tc.taskConfig.Project.Post != nil && tc.taskGroup == "" {
-		grip.Info("Running post-task commands")
```

Comment:
Was taking out this log statement intentional?

Reply:
It was. This gets logged locally, which is not a thing we check, so I think logging to Evergreen is sufficient.